### PR TITLE
fence_gce: Clean up and log errors

### PR DESCRIPTION
**Pull request by krig**
Catch any exceptions and write them to the log.

Also generally make the code a bit more pythonic.

### CODE REVIEW

**Comment by oalbrigt** [comment]
I dont think we should rewrite tabs to spaces for indentation, as long as the indentation is consistent within the agent that should be good enough, and you might cause new issues by changing it in Python code.

**Comment by krig** [comment]
alright, I’ll revert that.

**Comment by oalbrigt** [comment]
LGTM.

File: fence/agents/gce/fence_gce.py [hunk 1]
```diff
@@ -3,55 +3,57 @@
...
-				status = "on"
-			elif instance["status"] == "TERMINATED":
-				status = "off"
-			result[instance["id"]] = (instance["name"], status)
-	# TODO: check which Exceptions it can throw
-	except:
-		fail_usage("Failed: Unable to connect to GCE. Check your configuration.")
+			result[instance["id"]] = (instance["name"], translate_status(instance["status"]))
+	except Exception as err:
+		fail_usage("fence_gce.get_nodes_list: {}".format(str(err)))
```
Comment (oalbrigt) [inline fence/agents/gce/fence_gce.py]: I think these fail_usage() calls should still say Failed: and fence_gce would be provided by the logging facility, so e.g. ""Failed: get_nodes_list: {}" in this case.

**Pull request merged**